core/src/main/resources/hudson/PluginManager/available.jelly

```diff
@@ -52,36 +51,37 @@ THE SOFTWARE.
           </div>
         </div>
 
-        <table id="plugins" class="jenkins-table sortable"
-               data-hasAdmin="${h.hasPermission(app.ADMINISTER)}">
-          <thead>
-            <tr>
+        <form method="post" action="install">
+          <table id="plugins" class="jenkins-table sortable"
+                 data-hasAdmin="${h.hasPermission(app.ADMINISTER)}">
+            <thead>
+              <tr>
+                <l:isAdmin>
+                  <th data-sort-disable="true">
+                    ${%Install}
+                  </th>
+                </l:isAdmin>
+                <th initialSortDir="down">${%Name}</th>
+                <th>${%Released}</th>
+                <j:if test="${isUpdates}">
+                  <th>${%Installed}</th>
+                </j:if>
+              </tr>
+            </thead>
+            <tbody/>
+          </table>
+
+          <div id="bottom-sticker">
+            <div class="bottom-sticker-inner">
               <l:isAdmin>
-                <th data-sort-disable="true">
-                  ${%Install}
-                </th>
+                <f:submit value="${%Install without restart}"/>
```

Comment:
```suggestion
                <f:submit value="${%Install without restart}" name="dynamicLoad" />
```

This should be reverted: It changes the behavior of "Install without restart" to be just like "Download now and install after restart", i.e., every plugin installation will require a restart now for the plugin to load.